xpbooster.java

```diff
@@ -20,25 +20,32 @@ public class Xpbooster extends Plugin
 	@Inject
 	private Client client;
 
-	private static final String TOTAL_LEVEL_TEXT_PREFIX = "Total level:<br>";
+	private HashMap<Skill, Integer> skillToChildId = new HashMap<>();
+
+	private final String TOTAL_LEVEL_TEXT_PREFIX = "Total level:<br>";
 
 	HashMap<Skill, Integer> realXpTracker = new HashMap<>();
 	HashMap<Skill, Integer> fakeXpTracker = new HashMap<>();
 
 	void initialiseSkills(){
 		System.out.println("Initialising skills");
+		Integer currentSkillChildId = 1;
+
 		for (Skill skill : Skill.values()){
 			if (skill != Skill.OVERALL){
 				fakeXpTracker.put(skill, client.getSkillExperience(skill));
 				realXpTracker.put(skill, client.getSkillExperience(skill));
+
+				skillToChildId.put(skill, currentSkillChildId);
```

Comment:
This relies on Enum ordering. As far as I can see, the enum ordering is consistent with the switch statement, so this should work.